graph.go

```diff
@@ -402,13 +403,22 @@ func (g *Graph[K]) Add(nodes ...Node[K]) {
 
 		// Invariant check: the node should have been added to the graph.
 		if g.Len() != preLen+1 {
-			panic("node not added")
+			slog.Warn("node not added")
```

Comment:
Remove this log please. It's bad form for a data-structure library to take real estate in stdio.